docs/fflags.adoc

```diff
@@ -0,0 +1,23 @@
+== Feature Flags
+
+The following system properties are available for toggling or specifying particular features to turn on or off.
+Like any system properties, they should be considered experimental and subject to removal without notice.
+
+* `com.cloudbees.plugins.credentials.UseOwnPermission`
+  - Setting this property to `true` enables the _Credentials/UseOwn_ permission.
+    Primarily useful when using project-based security authorization strategies or authenticated builds, this permission allows an immediate action to access user-scoped credentials from the user's private credentials store.
+    Immediate actions include invoking a build with parameters, tagging a build in SCM, providing credentials to a pipeline input step, and other user interaction.
+    Using this permission can help limit which users are allowed to provide their own credentials as input to an action which can be particularly handy when combined with the Authorize Project plugin where builds may be run as the user who triggered the build.
+    This allows separation of the permission to build a job from the permission to provide user-scoped credentials.
+    When this flag is `false` or unspecified, then _Job/Build_ is used instead for the same purpose.
```

Comment:
As a personal preference, I want to see the default of something early, to better understand the context.

Reply:
I like that preference!